idtitlecreatedAtclosedAtclosed
MDU6SXNzdWUyMDQyNDUwODc=licensing2017-01-31T09:39:14Z2017-01-31T10:26:03ZTrue
MDU6SXNzdWUyMTU1MDcwMDU=SNMP Error when running 2017-03-20T18:04:11ZFalse
MDU6SXNzdWUyMjE4NTQ4ODQ="Overlay or Cross ""No data received"""2017-04-14T17:40:13ZFalse
MDU6SXNzdWUyMjU0MDA2NTU=Capture list is null2017-05-01T07:50:48ZFalse
MDU6SXNzdWUyMjU0MDM1NTI=printer.py self.cmdloop() function definition not find2017-05-01T08:22:12ZFalse
MDU6SXNzdWUyMjg4ODY5NDU=Running without arguments results in error instead of scan2017-05-16T01:01:41Z2017-05-16T01:03:21ZTrue
MDU6SXNzdWUyNDY4NjgyNjQ=SNMP Error (;ValueError: invalid literal for int() with base 10: '')2017-07-31T20:06:17ZFalse
MDU6SXNzdWUyNjA4MjY3Nzk=IPv6 ISSUE2017-09-27T02:26:29ZFalse
MDU6SXNzdWUyNjgzMjA0ODY=Syntax error running PRET from Termux on Android2017-10-25T09:07:11ZFalse
MDU6SXNzdWUyNzcwMzA4ODc=SNMP Error when running PRET2017-11-27T14:09:46ZFalse
MDU6SXNzdWUyODk5NjQ5NTM=Connection Failed2018-01-19T12:47:36ZFalse
MDU6SXNzdWUyOTIxNzA3NzU=Not working with local USB printer (Epson Stylus TX125)2018-01-28T02:23:52Z2018-01-28T11:49:25ZTrue
MDU6SXNzdWUzMjI1Nzc3NTk=Command execution failed (timed out)﻿ - HP 55402018-05-13T09:17:58Z2018-05-13T19:42:25ZTrue
MDU6SXNzdWUzMjI1OTc3MTc=Access USB printer on windows (Shared printer)2018-05-13T14:17:42ZFalse
MDU6SXNzdWUzMzM5OTQxNTg=crash 2018-06-20T09:25:26ZFalse
MDU6SXNzdWUzNDcwNTI5MzI=Active Directory Printer 2018-08-02T15:04:32ZFalse
MDU6SXNzdWUzNDcyMTQwNzg=Python3 2018-08-02T23:59:57ZFalse
MDU6SXNzdWUzNDk0NzE1NzU=Mac OSX support2018-08-10T10:58:13ZFalse
MDU6SXNzdWUzNTAzNzE1OTk=x2018-08-14T10:35:26Z2019-02-01T14:31:09ZTrue
MDU6SXNzdWUzODc0NTEzMzY=PJL ls directory traversal outputting wrong data2018-12-04T19:47:59Z2018-12-30T10:07:58ZTrue
MDU6SXNzdWU0MTA4NDAyOTI=HP printers device file2019-02-15T16:14:57ZFalse
MDU6SXNzdWU0MjQwMTM2MDA="I get an error in termux: ""Device: Receiving data failed (watchdo timeout)"""2019-03-22T01:54:33ZFalse
MDU6SXNzdWU0MjY3MDk2NjM=i can't do this, why idk2019-03-28T21:12:19Z2020-04-21T12:46:34ZTrue
MDU6SXNzdWU0MzA3MDc2NDk=Convert-im6.q16: no images defined `pcl:-'2019-04-08T23:58:17ZFalse
MDU6SXNzdWU0NjA2NzUyMzI=Requires a second connect for it to print when using --load2019-06-25T22:32:26ZFalse
MDU6SXNzdWU0NzAxMDIzNzQ=ConvertProgram Error!2019-07-19T02:07:38ZFalse
MDU6SXNzdWU0ODI3MTYzNTg="""This only affects jobs printed by a PostScript driver"" ?"2019-08-20T08:25:52ZFalse
MDU6SXNzdWU1MDA5NDU5NTE=Connection Closed / Timed out2019-10-01T14:43:57ZFalse
MDU6SXNzdWU1MDU5NDc0MzU=Any chance to get this to work on Python 3?2019-10-11T16:29:03ZFalse
MDU6SXNzdWU1MTU1NzkwMzM=Issues with printers that have kernel loading issues . [Xerox WorkCentre type] 2019-10-31T16:37:30ZFalse
MDU6SXNzdWU1ODQwMjQ5MzE=does not print2020-03-18T21:51:19ZFalse
MDU6SXNzdWU2MTg0NTU5OTc=HP LaserJet - Out of memory, will not capture jobs anymore2020-05-14T18:43:47ZFalse
